### ISSUE TITLE
Exists property is missing from ExistsResponse and ExistsAliasResponse

### ISSUE THREAD

**Issue by gpetrou** [elastic/elasticsearch-net#8082]
**Elastic.Clients.Elasticsearch version**: 8.13.1

**Elasticsearch version**: 8.11

**.NET runtime version**: 8.0

**Operating system version**:

**Description of the problem including expected versus actual behavior**:
`ExistsResponse` (both classes) and `ExistsAliasResponse` do not have an `Exists` property anymore.

**Steps to reproduce**:
 1.
 2.
 3.

**Expected behavior**

**Provide `ConnectionSettings` (if relevant)**:

**Provide `DebugInformation` (if relevant)**:

**Comment by flobernd** [comment]
Hi @gpetrou,

thanks for reporting!

These properties are not defined in the specification. The old generator had a hardcoded condition to emit these properties. I'll manually add them back in the next patch release.

**Comment by TheFireCookie** [comment]
Do we need them? I think that "IsValidResponse" is enough to validate that, no?

**Comment by flobernd** [comment]
@TheFireCookie That's correct, we don't really need them. These properties are just checking the status code - just like `IsValidResponse`:

```csharp
public bool Exists => ApiCallDetails is
{
    HasSuccessfulStatusCode: true, HttpStatusCode: 200
};
```

For now I'll still add them back to not introduce more breaking changes than needed 🙂 

I might remove them in the next major version. Another option would be to keep them and override `IsValidResponse` in addition. For these endpoints, a 404 is actually valid in my opinion. But that might over-complicate things as users are probably used to expect `false` for `IsValidResponse` when entities are non existing.

**Comment by gpetrou** [comment]
This property exists since NEST. Why remove it? There are already too many breaking changes to worry about.

**Comment by flobernd** [comment]
Fixed in 8.13.4
